src/org/labkey/test/components/domain/DomainFieldRow.java

```diff
@@ -505,6 +506,12 @@ public DomainFieldRow setConceptImportField(String importField)
         return this;
     }
 
+    public List<String> getConceptImportSelectOptions()
```

Comment:
This change enables the test to get which ontologies are available/selectable in the field row